test/tests/httptests.ts

```diff
@@ -118,9 +117,9 @@ describe('Http Tests', function () {
 
     it('pipes a get request', () => {
         return new Promise<string>(async (resolve, reject) => {
-            let file: NodeJS.WritableStream = fs.createWriteStream(sampleFilePath);
-            (await _http.get('https://httpbin.org/get')).message.pipe(file).on('close', () => {
-                let body: string = fs.readFileSync(sampleFilePath).toString();
+            let file = new streamBuffers.WritableStreamBuffer();
```

Comment:
Nit - add type

Reply:
Also, is there a reason to change this? Seems out of scope of the current PR and I'm not really sure what the purpose is